packages/mask/src/plugins/CryptoartAI/constants.ts

```diff
@@ -0,0 +1,36 @@
+import { ChainId } from '@masknet/web3-shared-evm'
+
+export const PLUGIN_NAME = 'CRYPTOART.AI'
+export const PLUGIN_ICON = '🖼️'
+export const PLUGIN_DESCRIPTION = 'The decentralized world of CryptoArt.'
+export const PLUGIN_IDENTIFIER = 'com.maskbook.cryptoartai'
+export const PLUGIN_META_KEY = 'com.maskbook.cryptoartai:1'
+
+export const cryptoartaiContractAddress = [
+    '0x3AD503084f1bD8d15A7F5EbE7A038C064e1E3Fa1',
+    '0x54395e6c737734D3de29Fc62C10a3ed51eFA2E8A',
+]
+
+export const cryptoartaiHostnames = ['cryptoart.ai', 'testweb.cryptoart.ai']
+export const cryptoartaiPathnameRegexMatcher = '/gallery/detail/'
+
+export const cryptoartaiAPIEndpoint = ['https://api.cryptoart.ai', 'https://apitest.cryptoart.ai']
```

Comment:
plz redesign the definition.

`cryptoartai` variable/constant prefix name is meaningless.

e.g:
```suggestion
const mainNetwork = {
    hostname: 'testweb.cryptoart.ai',
    endpoint: 'https://apitest.cryptoart.ai',
}

const testNetwork = {
    hostname: 'cryptoart.ai',
    endpoint: 'https://api.cryptoart.ai',
}

export const pathnameRegexMatcher = '/gallery/detail/'
```

Reply:
Thanks, improved.